### PR TITLE
[TG-63][feat] Back end for list of scenarios

### PR DESCRIPTION
**Pull request by lizavetaageenko**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-backend/pull/7" target="_blank">ReviewNinja

File: src/API/scenarios/scenario.model.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import Promise from 'bluebird';
```
Comment (yurii-sorokin) [inline src/API/scenarios/scenario.model.js]: Why we use bluebird? Mongoose support native promises

Comment (quasiunafantasia) [inline src/API/scenarios/scenario.model.js]: we promisify mongoose with bluebird. Bluebird promises are used here to keep promise API the same (for example [.error ](http://bluebirdjs.com/docs/api/error.html#error) method), so we can always expect a bluebird promise from db operations.

File: src/config/seed.js [hunk 2]
```diff
@@ -21,10 +24,33 @@ sensors.push(new Sensor({
     mqttId: '2'
 }));
 
+scenarios.push(new Scenario({
+    name: 'some name',
+    description: 'some description',
+    body: 'alert(\'hello\');'
```
Comment (kosich) [inline src/config/seed.js]: really?

**Comment by kosich** [comment]
👍

**Pull request merged**